extras/rya.console/src/main/java/mvm/rya/shell/command/mongo/administrative/MongoUninstall.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package mvm.rya.shell.command.mongo.administrative;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import com.mongodb.MongoClient;
+
+import mvm.rya.shell.command.CommandException;
+import mvm.rya.shell.command.InstanceDoesNotExistException;
+import mvm.rya.shell.command.administrative.Uninstall;
+import mvm.rya.shell.command.mongo.MongoCommand;
+import mvm.rya.shell.command.mongo.MongoConnectionDetails;
+
+// TODO impl, test
+
+/**
+ * An Mongo implementation of the {@link Uninstall} command.
+ */
+@ParametersAreNonnullByDefault
+public class MongoUninstall extends MongoCommand implements Uninstall {
```

Comment:
This seems easy to implement on the Mongo side since you just drop a DB, so I think we should support it.